fiftyone/core/tags.py

```
@@ -1114,19 +1199,23 @@ def _build_update_fields(
+            index_type=doc["index_type"],
+            anchor=doc.get("anchor", None),
+            start=doc["start"],
+            end=doc["end"],
+            tag=doc["tag"],
+            created_by=doc.get("created_by", None),
+            last_modified_by=doc.get("last_modified_by", None),
+            created_at=doc.get("created_at", None),
+            last_modified_at=doc.get("last_modified_at", None),
+        )
+    raise ValueError(
+        "Expected a temporal tag document, but found kind=%s" % kind
```

Ruff (0.15.21)

[warning] Use format specifiers instead of percent format

Replace with format specifiers

(UP031)